example_scripts/SEAsia_R12_example_plot.py

```diff
@@ -20,15 +20,15 @@
 dir_nam = "/projectsa/COAsT/NEMO_example_data/SEAsia_R12/"
 fil_nam = "SEAsia_R12_5d_20120101_20121231_gridT.nc"
 dom_nam = "domain_cfg_ORCA12_adj.nc"
+config_t = "/work/jelt/GitHub/COAsT/example_files/example_t_nemo_config.json"
```

Comment:
Before this gets merged, can we change this to something generic (i.e. using "~") and platform-independent (i.e. using Path)?